widget/embedded/src/hooks/useWalletList.ts

```diff
@@ -55,9 +55,12 @@ export function useWalletList(params: Params) {
   );
 
   wallets = detectMobileScreens()
-    ? wallets.filter(
-        (wallet) => wallet.showOnMobile || state(wallet.type).installed
-      )
+    ? wallets.filter((wallet) => {
+        if (wallet.showOnMobile === false) {
+          return false;
+        }
+        return wallet.showOnMobile || state(wallet.type).installed;
```

Comment:
Can we remove `wallet.showOnMobile` in ` wallet.showOnMobile || state(wallet.type).installed` condition?

because in prev condition, `wallet.showOnMobile` was checked.

Reply:
The issue is that although showOnMobile might include the true value, but it isn't installed.

Reply:
in this condition: `wallet.showOnMobile || state(wallet.type).installed`,
`wallet.showOnMobile `  is always `true`,
so we can remove it or combine the two conditions as follows:

`wallets.filter(wallet => wallet.showOnMobile !== false && state(wallet.type).installed`